Ecosystem Partners/getting-started-with-drupal-blueprint.md

```diff
@@ -11,7 +11,7 @@
 ### Description
 
 <img src="//d33np9n32j53g7.cloudfront.net/assets/stacks/drupal/img/drupal-stack-110x117-9083fe6b191b13f79863eff74d3ce382.png" style="border:0;float:right;max-width:250px">
```

Comment:
That image link works for me.  Can you reconfirm the issue?  Or is there a translation the tooling requires the protocol in the URL?